network/calico/calico-k8s-backend-metallb.yaml

```
@@ -615,8 +624,8 @@ spec:
             # The default IPv4 pool to create on startup if none exists. Pod IPs will be
             # chosen from this range. Changing this value after installation will have
             # no effect. This should fall within `--cluster-cidr`.
-            - name: CALICO_IPV4POOL_CIDR
-              value: "192.168.0.0/16"
+            # - name: CALICO_IPV4POOL_CIDR
+            #   value: "192.168.0.0/16"
```

We want to use the `172.16.0.0/12` subnet for our pods subnet.
```suggestion
            - name: CALICO_IPV4POOL_CIDR
              value: "172.16.0.0/12"
```

---

@pereztr5 Doesn't calico use the CIDR specified by kubeadm if nothing is specified in calico?

Referring to this line:

https://github.com/enkelprifti98/kubernetes-bgp/blob/514a70579cf8b426454867cdda26974621c1aeef/templates/setup-kubeadm.sh.tpl#L53

---

It looks like in the master branch we have been specifying the service CIDR there...

https://github.com/packet-labs/kubernetes-bgp/blob/91b4c6d291af8e857e25aa43cee4370308b02f7a/network/calico/calico-k8s-backend-metallb.yaml#L618

---

Sounds good. Thanks for confirming.